tests/test_callbacks.py

```diff
@@ -101,44 +79,11 @@ def test_entropy_switch_partition(backend):
     backend.assert_allclose(result, 1.0)
 
 
-def test_entropy_numerical(backend):
-    """Check that entropy calculation does not fail for tiny eigenvalues."""
-    eigvals = np.array(
-        [
-            -1e-10,
-            -1e-15,
-            -2e-17,
-            -1e-18,
-            -5e-60,
-            1e-48,
-            4e-32,
-            5e-14,
-            1e-14,
-            9.9e-13,
-            9e-13,
-            5e-13,
-            1e-13,
-            1e-12,
-            1e-11,
-            1e-10,
-            1e-9,
-            1e-7,
-            1,
-            4,
-            10,
```

Comment:
I remember in the past we added this test for weird values because we were getting some failures. Maybe it is still useful to test if the new calculation works with them (if you are not already doing it)?

Reply:
The `quantum_info.entropy` function already takes care of these cases internally by explicitly setting to 0.0 the values where `log` isn't defined.